requirements.txt

```diff
@@ -1,4 +1,4 @@
 rdflib
 dominate
 matplotlib
-
+ddt==1.1.1
```

Comment:
If ddt is only required for the tests, let's not add it as a dependency for the library. We can document in the README that ddt is required to run the test and add `pip install ddt` to `.travis.yml`.